### ISSUE TITLE
Build failes on (Void) Linux with make

### ISSUE THREAD

**Issue by fluentpwn** [urho3d/urho3d#3158]
Building with make,

Process:
```
[  0%] Built target rapidjson
[  0%] Built target STB
[  1%] Built target ETCPACK
[  1%] Built target LZ4
[  2%] Built target StanHull
[  2%] Built target Civetweb
[  2%] Built target PugiXml
[  4%] Built target ik
[  4%] Built target DetourTileCache
[  5%] Built target Detour
[  5%] Built target DetourCrowd
[  5%] Built target Recast
[  5%] Built target GLEW
[  8%] Built target AngelScript
[ 11%] Built target FreeType
[ 14%] Built target Box2D
[ 22%] Built target WebP
[ 30%] Built target SLikeNet
[ 42%] Built target SDL
[ 54%] Built target Bullet
[ 74%] Built target Urho3D
[ 75%] Linking CXX executable ../../../bin/02_HelloGUI
[ 75%] Linking CXX executable ../../../bin/tool/PackageTool
[ 75%] Linking CXX executable ../../../bin/tool/ScriptCompiler
[ 75%] Linking CXX executable ../../../bin/03_Sprites
[ 75%] Linking CXX executable ../../../bin/01_HelloWorld
[ 75%] Linking CXX executable ../../../bin/tool/RampGenerator
[ 75%] Linking CXX executable ../../../bin/Urho3DPlayer
[ 75%] Linking CXX executable ../../../bin/tool/Tests
[ 75%] Linking CXX executable ../../../bin/04_StaticScene
[ 75%] Linking CXX executable ../../../bin/tool/OgreImporter
[ 75%] Linking CXX executable ../../../bin/tool/SpritePacker
[ 91%] Built target Assimp
```

Error:
```
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `Wayland_VideoInit':
SDL_waylandvideo.c:(.text+0x5fe): undefined reference to `wl_proxy_get_version'
/bin/ld: SDL_waylandvideo.c:(.text+0x61c): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `display_handle_global':
SDL_waylandvideo.c:(.text+0x86b): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: SDL_waylandvideo.c:(.text+0x8ca): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: SDL_waylandvideo.c:(.text+0x93b): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: SDL_waylandvideo.c:(.text+0x979): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o):SDL_waylandvideo.c:(.text+0x9d9): more undefined references to `wl_proxy_marshal_flags' follow
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `handle_ping_xdg_wm_base':
SDL_waylandvideo.c:(.text+0xb5f): undefined reference to `wl_proxy_get_version'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `handle_ping_zxdg_shell':
SDL_waylandvideo.c:(.text+0xb9f): undefined reference to `wl_proxy_get_version'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `Wayland_VideoQuit':
SDL_waylandvideo.c:(.text+0xd04): undefined reference to `wl_proxy_get_version'
/bin/ld: SDL_waylandvideo.c:(.text+0xd1a): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: SDL_waylandvideo.c:(.text+0xd2c): undefined reference to `wl_proxy_get_version'
/bin/ld: SDL_waylandvideo.c:(.text+0xd42): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `handle_ping_xdg_wm_base':
SDL_waylandvideo.c:(.text+0xb7e): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandvideo.c.o): in function `handle_ping_zxdg_shell':
SDL_waylandvideo.c:(.text+0xbbe): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: ../../../lib/libUrho3D.a(SDL_waylandopengles.c.o): in function `Wayland_GLES_SwapWindow':
SDL_waylandopengles.c:(.text+0x108): undefined reference to `wl_proxy_get_version'
/bin/ld: SDL_waylandopengles.c:(.text+0x128): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: SDL_waylandopengles.c:(.text+0x183): undefined reference to `wl_proxy_get_version'
/bin/ld: SDL_waylandopengles.c:(.text+0x19c): undefined reference to `wl_proxy_marshal_flags'
/bin/ld: SDL_waylandopengles.c:(.text+0x1b3): undefined reference to `wl_proxy_get_version'
...
```
Overall error output is 3027 lines long so this is a stub, errors are all of the same sort.
`LDFLAGS="${LDFLAGS} -lwayland-client"` doesn't change anything.

Linux kernel ver.: 6.0.11 x86_64
Wayland libs ver.: 1.21 / latest git
Distro: Void

**Comment by 1vanK** [comment]
Wayland is not supported yet

p.s. I have nVidia and Linux Mint. And Wayland does not work for me in any applications.

**Comment by 1vanK** [comment]
https://forums.developer.nvidia.com/t/wayland-information-for-r515-beta-release/214275

**Comment by fluentpwn** [comment]
> Wayland is not supported yet
> 
> p.s. I have nVidia and Linux Mint. And Wayland does not work for me in any applications.

Then, how can I compile Urho3D with just the X11 support?

**Comment by 1vanK** [comment]
Fixed in https://github.com/urho3d/urho3d/pull/3163

Reopen if it doesn't work

**Comment by fluentpwn** [comment]
> Fixed in #3163
> 
> Reopen if it doesn't work

Sorry for no response, I changed my distro to artix recently and had no time. Yes, now build does work(or at least on Artix it does). Thanks!
